data/remnant missions.txt

```diff
@@ -495,3 +560,703 @@ mission "Remnant: Technology Available"
+			`	Tali sits over at a corner booth with the prefect who met you at the dock and a couple other mechanics, and she waves you over. "Captain <last>! So, how do you like fighting with our fleet?" asks one of the mechanics.`
+			choice
+				`	"It seems odd to have a planetary defense fleet plundering."`
+				`	"It was a tough battle."`
+					goto next
+	
+			`	"I suppose it would," chants one of them. "We need access to their tech, though. Otherwise we'll fall behind." Judging by the finality of his statement, this is held to be common knowledge among the Remnant.`
+				goto next
+	
+			label next
+			`	While you're conversing with the group, the prefect suddenly glances down at his comlink, reads a message, then looks up at you. "It appears that a Korath ship survived the raid and has destroyed a few ships without taking cargo. Could you deal with it?"`
+			choice
+				`	"Sure."`
+				`	"Sorry, I don't have time to help you out."`
+					decline
+
+			`	"Thank you," he signs. "It is acting differently, jumping around destroying ships instead of looting and running. If it simply looted and left we would be pleased, but now it is learning about us instead, so it has reached the end of its usefulness. Please disable it so that our technicians can get on board." He pauses, then continues, "I doubt we'll be able to salvage the ship itself - they pack those things with soldiers - but we should be able to strip everything else of use off it."`
+				accept
+	npc save disable
+		government Korath
+		personality heroic vindictive target uninterested waiting
+		system
+			distance 1 2
+		fleet
+			names "korath"
+			cargo 3
+			variant
+				"Korath Raider (Hyperdrive)"
+		dialog "You have disabled the Korath ship that was left over from the raid on <planet>. You can now return there to collect your payment."
+	on fail
+		payment 250000
+		dialog `You are met by the prefect who thanks who looks upset. "The ship was supposed to be disabled so its contents could be salvaged!" he signs angrily. Nonetheless, he shoves a credit chip worth <payment> into your hand. "That's half what you would have earned if the ship had been disabled." He pauses and shakes his head. "I think Tali might have some more work for you in the spaceport, but in the meantime, why don't you go find some Korath to practice disabling? It is a valuable skill among the Remnant."`
+	on complete
+		payment 500000
+		dialog `You are met by the prefect who thanks you for disabling it and hands you a credit chip for <payment>. "I think Tali might have some more work for you in the spaceport, but in the meantime, don't forget to drop off any salvaged outfits in the outfitters so the researchers can look at them."`
+
+
+
+mission "Remnant: Bounty 2"
```

Comment:
I'd move the location of this bounty in the file to after the salvage missions, just so there isn't a break in the file for where all the salvage missions are.

Reply:
Moved.